src/scenes/Home/index.tsx

```diff
@@ -1,14 +1,9 @@
 import React from 'react';
-import { useSession } from '../../components/Session';
 import { useTitle } from '../../components/title';
+import { Header } from '../Header/Header/Header';
 
 export const Home = () => {
   useTitle('Home');
-  const [session] = useSession();
 
-  return (
-    <div>
-      <div>Welcome, {session?.displayFirstName.value ?? 'Friend'}</div>
-    </div>
-  );
+  return <Header />;
```

Comment:
Instead of making changes here, let's apply the `Header` in `Root`.

Reply:
Ok cool - what will go in here then?